searchlib/src/tests/fef/termmatchdatamerger/termmatchdatamerger_test.cpp

```diff
@@ -245,5 +245,98 @@ TEST(TermMatchDataMergerTest, merge_max_element_length)
     EXPECT_EQ(1000u, out.getIterator().getFieldLength());
 }
 
+TEST(TermMatchDataMergerTest, merge_no_normal_features)
+{
+    TermFieldMatchData a;
```

Comment:
These 3 tests share basically the same setup, from line 250 to 260. The only difference is which bools are set on **_out_**. I suggest putting the shared code into a test class with an init() function (or similar) that takes the two bools and instantiate the merger.